src/test/java/core/basesyntax/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,58 @@
+package core.basesyntax;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.dto.Transaction;
+import core.basesyntax.model.Fruit;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+    private static OperationHandler handler;
+
+    @BeforeClass
+    public static void beforeClass() {
+        handler = new BalanceOperationHandler();
+    }
+
+    @Before
+    public void setUp() {
+        Storage.getStorage().put(new Fruit("apple"), 45);
+        Storage.getStorage().put(new Fruit("banana"), 77);
+    }
+
+    @Test
+    public void balanceHandler_addTransactionWithExistingFruit_Ok() {
+        Transaction transaction = new Transaction("b", "apple", 100);
+        handler.perform(transaction);
+        int actual = Storage.getStorage().get(new Fruit("apple"));
```

Comment:
якщо створиш константу уникнеш створювання ідентичних об'єктів
```suggestion
        int actual = Storage.getStorage().get(APPLE);
```